lib/src/analyzers/unused_files_analyzer/unused_files_analyzer.dart

```diff
@@ -50,8 +50,13 @@ class UnusedFilesAnalyzer {
       final unusedFilesAnalysisConfig =
           ConfigBuilder.getUnusedFilesConfig(contextConfig, rootFolder);
 
+      final contextFolders = folders
+          .where((path) => normalize(join(rootFolder, path))
```

Comment:
@incendial looks similar than code at 37 line. And what about similar bloc at lib/src/analyzers/lint_analyzer/lint_analyzer.dart ?